LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -768,6 +768,37 @@ std::wstring etwJsonFormat(EtwLogEntry* pLogEntry)
     return oss.str();
 }
 
+///
+/// Format ETW eventlog into a XML output
+///
+std::wstring etwXMLFormat(EtwLogEntry* pLogEntry)
+{
+    std::wostringstream oss;
+
+    // construct the JSON output

```

Comment:
nit: misleading comment, should be `XML`

Reply:
Commenting change done